.github/workflows/contrib-tests.yml

```diff
@@ -0,0 +1,47 @@
+name: Contrib tests
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - ci_contrib_*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        contrib: [
+          "sentence_transformers",
+          "timm",
+        ]
```

Comment:
Great idea to use a matrix here!

Reply:
Should this matrix be governed by a `ls` first so that we have individual jobs for each folder under `contrib` and without the need to specify each folder here? (nitpick though, this should (or shouldn't) be done in a follow up PR)

Reply:
Added in https://github.com/huggingface/huggingface_hub/pull/1200/commits/f20ab77919533fa1254d4e89df3884956644c259 and https://github.com/huggingface/huggingface_hub/pull/1200/commits/6cb96e7658c72922dfb5cf9b7b7773e0df5fcf53 a script that list contrib tests and update the Makefile and github workflow file accordingly. Script is integrated with `make quality` and `make style` to make it easy for contributors.